modules/manage/partials/iceberg/use-iceberg-catalogs.adoc

```diff
@@ -39,9 +38,109 @@ For REST catalogs that use self-signed certificates, also configure these proper
 
 See xref:reference:properties/cluster-properties.adoc[Cluster Configuration Properties] for the full list of cluster properties to configure for a catalog integration.
 
+ifdef::env-cloud[]
+=== Store a secret for REST authentication
+
+To store a secret that you can reference in your catalog authentication cluster properties, you must create the secret using `rpk` or the Data Plane API. Secrets are stored in the secret management solution of your cloud provider. Redpanda retrieves the secrets at runtime.
+
+Store secrets for the following properties:
+* `iceberg_rest_catalog_client_secret`
+* `iceberg_rest_catalog_crl`
+* `iceberg_rest_catalog_token`
+* `iceberg_rest_catalog_trust`
+
```

Comment:
The live version of this page currently specify `iceberg_rest_catalog_crl_file` and `iceberg_rest_catalog_trust_file`: https://docs.redpanda.com/current/manage/iceberg/use-iceberg-catalogs/#connect-to-a-rest-catalog Should we change those to `_crl` and `_trust` instead?

Reply:
we cannot use the ones with _file in cloud because you don't have access to filesystem